manual/php.md

```diff
@@ -47,6 +47,8 @@ For all control structures there is a space between the keyword and an opening p
 
 For all all control structures, such as `if`, `else`, `do`, `for`, `foreach`, `try`, `catch`, `switch` and `while`, both the keyword starts a newline and the opening and closing braces are each put on a new line.
 
+Exclamation mark `!` logical operator `not` used a condition should not have spaces before or after the exclamation mark as shown in the examples.
```

Comment:
Used in a condition?

Even with that change I’m not sure this reads right though. I’m not quite following what the “not” is about

Reply:
Reading it again, I agree the grammar is a little odd there. I was trying to explain the operator briefly for the new coders or non-coders who also contribute and have to comply with the coding standard. 

Maybe something like this?

Exclamation mark `!`, (i.e. the logical operator `not`), used a condition should not have spaces before or after the exclamation mark as shown in the examples.

Reply:
I agree with George, it should read "used *in* a condition"

Reply:
I made a change. I hope that sentence makes more sense now.